src/contact.ts

```diff
@@ -232,6 +234,10 @@ export class Contact implements Sayable {
       query = { name: /.*/ }
     }
 
+    if (query[0] === 'remark') {
+      log.warn('Contact', 'findAll(remark:%s) DEPRECATED, use findAll(alias:%s) instead.')
```

Comment:
no, you misunderstand.

you should remove `remark` in keyMap first, then you will have to follow my request.